quickstart.md

```diff
@@ -406,3 +281,136 @@ To monitor your validator process, use the following pm2 commands to monitor the
 pm2 status
 pm2 logs <id>
 ```
+
+
+## Other Setup (Reference Only in This Version):
```

Comment:
Maybe add a few comments indicating "this task is no longer being distributed in the current version. enable this setup only if you need to support vector retrieval tasks"

Reply:
solved by a new update